heka/sandbox/decoders/extract_executive_summary.lua

```diff
@@ -0,0 +1,176 @@
+-- This Source Code Form is subject to the terms of the Mozilla Public
+-- License, v. 2.0. If a copy of the MPL was not distributed with this
+-- file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+--[[
+Convert the massive unified telemetry submission into the small sub-set of data
+required to power the executive dashboard.
+
+See: https://bugzilla.mozilla.org/show_bug.cgi?id=1155871
+
+*Example Heka Configuration*
+
+.. code-block:: ini
+
+    [TelemetryInput]
+    # File or S3 input
+    decoder = "MultiDecoder"
+
+    [MultiDecoder]
+    subs = ["ProtobufDecoder", "ExecutiveSummary"] # use for S3 telemetry data
+    #subs = ["TelemetryDecoder", "ExecutiveSummary"] # use for S3 landfill data
+    cascade_strategy = "all"
+    log_sub_errors = true
+
+    [TelemetryDecoder]
+    type = "SandboxDecoder"
+    filename = "lua_decoders/extract_telemetry_dimensions.lua"
+    memory_limit = 100000000
+    output_limit = 1000000
+
+    [ExecutiveSummary]
+    type = "SandboxDecoder"
+    filename = "lua_decoders/extract_executive_summary.lua"
+    memory_limit = 100000000
+--]]
+
+require "cjson"
+local l = require "lpeg"
+require "string"
+
+local SEC_IN_HOUR = 60 * 60
+local SEC_IN_DAY = SEC_IN_HOUR * 24
```

Comment:
They are more global than fx.lua but I am not sure I am sold on a constants module since in most cases you would pull in more than you need.